### ISSUE TITLE
Multiple concurrent folder syncs and/or sync prioritization

### ISSUE THREAD

**Issue by Webbeh** [nextcloud/desktop#2376]
I'm pretty sure I'm not the only one who has this issue. 

On the same NC server, I'm using multiple NextCloud accounts, and each of those have multiple folder syncs.
One of those syncs is a massive video folder where I upload all my GoPro reels, about 200GB of data per day.

### Actual behaviour
Any huge folder sync on any account will block every other account/folder synchronization from happening. This lasts until the huge folder is fully synchronized or manually paused.

### Expected behaviour
Either : 
- A concurrent folder synchronization, where multiple accounts and folder syncs would be able to work in parallel.
- A prioritization of syncs, where you could reorder, in the client, which folder has priority over others. If that folder is syncing, the others are paused.

### Steps to reproduce
1. Create two or more folder synchronizations
2. Place a massive file in one of them (several gigabytes)
3. Check the client to see that the other folder syncs are not happening while that big file is being synchronized

### Client configuration
Client version: Any 2.* version. Untested in 3.*

Operating system: Any

Qt version used by client package (Linux only, see also Settings dialog): Irrelevant

Client package (From Nextcloud or distro) (Linux only): Irrelevant

Installation path of client: Irrelevant

### Server configuration
Version : 19.0.2

### Logs
Irrelevant.

**Comment by Frizou** [comment]
See another suggestion here: https://github.com/nextcloud/desktop/issues/2452

**Comment by Rello** [comment]
Hello,

thank you for opening the request.
The client has improved a lot since the ticket was opened.
Can you support us and check if it is still relevant? In this case we appreciate a new ticket or a comment here.
Thank you for your support

**Comment by Webbeh** [comment]
Good morning. This is still relevant.


In this case, I started by sending a 1.5GB file to "Documents" (the one currently uploading), and then copied 2-3 pictures (< 5MB each) to the "Pictures" folder.
Here's what it looks like right after I copy the pictures to the Pictures folder : 
![Image](https://github.com/user-attachments/assets/045b3317-5496-4071-a9a6-33523ac95474)

Here's what it looks like a few minutes later : 
![Image](https://github.com/user-attachments/assets/afb86413-4656-49a9-9d7e-0dc7e352f6bc)


The Pictures folder sync now has to wait for the big upload to be done, to fail or to be paused. This happens everytime a big file is copied.

I can understand that, in the same folder sync (inside Documents), the files are treated one after the other. However, I cannot understand why two different folder syncs can't be executed in parallel.

I am using the default settings regarding upload, proxy, network settings, etc.

**Comment by JonathanTreffler** [comment]
This issue/feature request is still relevant (tested on Linux, I encounter this problem pretty much daily), so can it please be re-opened ?

**Comment by Webbeh** [comment]
I do believe this was extremely easy to test if it was indeed working or not, and @Rello 's message is very unsatisfactory at best.

As your message doesn't really match the "not planned" closing reason, please confirm this will indeed not be planned.
Thank you
